### ISSUE TITLE
8416 Staging - Update expense summary pages

### ISSUE THREAD

**Issue by thejordanwood** [department-of-veterans-affairs/va.gov-team#122491]
## Description
One ticket for all the changes we want to make to each summary page post MVP. 

**Care expense**

- Summary header
     - Currently displays in-home care attendant or residential care facility
     - Instead display the input entered into the "What’s the name of the care provider?" field
     - Define character limits
- Dates
     - Add start and end dates

<img width="429" height="242" alt="Image" src="https://github.com/user-attachments/assets/93a8bcfc-c7fd-4492-b1b3-9b62cd95b765" />

<details><summary>Completed expense types</summary>
<p>
**Medical expense**

- Summary header
     - Header is correct, should be Payment recipient
     - Define character limits
- Payment date is correct
- Add frequency of payment

<img width="545" height="387" alt="Image" src="https://github.com/user-attachments/assets/e4501c11-496b-480c-a392-1e98ab429b40" />

**Mileage**

- Summary header
     - Currently displays Other
     - Instead display the input entered into the "Tell us where you traveled" field
     - Define character limits

<img width="605" height="341" alt="Image" src="https://github.com/user-attachments/assets/94f0de87-324b-44ff-a1b5-2b2b4940d789" />
</p>
</details> 

### Acceptance criteria

- [ ] **Update care expense summary**
- [x] Update medical expense summary
- [x] Update mileage expense summary

**Comment by meaghan-sullivan** [comment]
Expense summaries updated for Medical Expenses: 

<img width="326" height="287" alt="Image" src="https://github.com/user-attachments/assets/f06106e3-a67f-4b1d-9bd4-9d32bf3aac3b" />

**Comment by meaghan-sullivan** [comment]
[ ] Care expenses: Does **not** display provider name
[X] Medical expenses: Does display payment recipient name, frequency
[X] Mileage expenses: Does display location, travel date


<img width="201" height="144" alt="Image" src="https://github.com/user-attachments/assets/5b308fe8-4f4c-4c75-ae6d-17f45cc641c2" />
<br>
<img width="203" height="98" alt="Image" src="https://github.com/user-attachments/assets/cd4a08b6-4c06-4da5-accd-ad9a71962c30" />
<br>
<img width="216" height="86" alt="Image" src="https://github.com/user-attachments/assets/f30f0ebf-c553-48ff-888c-7944ce28d90f" />

**Comment by kaise-lafrai** [comment]
#122365

**Comment by kaise-lafrai** [comment]
@thejordanwood What do you expect the date range in the care expense to look like if the user only enters the required start date and no end date. Would it say 12/01/2010 - N/A or 12/01/2010 - no end date or just 12/01/2010. Thanks!

**Comment by meaghan-sullivan** [comment]
Confirmed in Staging!
